aes/tests/mod.rs

```diff
@@ -1,6 +1,7 @@
 //! Test vectors are from NESSIE:
 //! https://www.cosic.esat.kuleuven.be/nessie/testvectors/
 
-cipher::block_cipher_test!(aes128_test, "aes128", aes::Aes128);
-cipher::block_cipher_test!(aes192_test, "aes192", aes::Aes192);
-cipher::block_cipher_test!(aes256_test, "aes256", aes::Aes256);
+// TODO(tarcieri): update tests to new format
+//cipher::block_cipher_test!(aes128_test, "aes128", aes::Aes128);
+//cipher::block_cipher_test!(aes192_test, "aes192", aes::Aes192);
+//cipher::block_cipher_test!(aes256_test, "aes256", aes::Aes256);
```

Comment:
@newpavlov hopefully this sort of thing is okay temporarily to unblock the `hybrid-array` upgrade

Reply:
It's fine, but could you explain why the tests do not work? It's a bit surprising to me.

Reply:
Ah, I forgot to migrate the tests? I can prepare a migration PR tomorrow. You can wait for it or merge this PR as-is.

Reply:
Added a reference, will merge when green. Yeah, looks like this repo didn't get updated.